### DISCUSSION TITLE
After installing errors in logs

### DISCUSSION THREAD

**Discussion by Oxize** [aaronwmorris/indi-allsky#917] [General]
After i was done installing i get errors in log. Ive didnt managed get any image yet.

Camera: HQ 
RPI: 4B 8GB

![image](https://github.com/aaronwmorris/indi-allsky/assets/3933702/eb951d0f-0514-4464-be82-92d4489e91e0)

how to fix?

**Comment by aaronwmorris** [comment]
I am investigating.  Do you have a GPS device configured?

**Reply by Oxize** [reply]
@aaronwmorris i selected gpsd, but i forgot i dont have place to connect any usb when its in his allysky housing. So, it need to be disabled somewhere.

And even by rerunning setup.sh, and putting gps on none, doesnt remove this error.

**Reply by aaronwmorris** [reply]
After you re-run setup.sh, you will need to manually restart the indiserver service to remove the indi gps server setup.

```
systemctl --user restart indiserver
```
